components/layout/Header.tsx

```diff
@@ -1,23 +1,41 @@
 import stls from '@/styles/components/layout/Header.module.sass'
-import { TypeClassNames } from '@/types/index'
+import { TypeGeneralClassNames } from '@/types/index'
+import { phoneNumber } from '@/config/index'
 import cn from 'classnames'
 import { getClassNames } from '@/helpers/index'
-import { HeaderTop, HeaderMenu, HeaderModileMenu } from '@/components/header'
+import { HeaderMenuDesktop, HeaderMenuMobile } from '@/components/header'
+import { Wrapper } from '@/components/layout'
+import { GeneralContactCTA, GeneralLogo } from '@/components/general'
+import { IconGeneralHamburger } from '@/components/icons'
+import { ImgGeneralGMDELogo, ImgGeneralRABOLogo } from '@/components/images'
 
+import { useState } from 'react'//test
 
 
 
-type TypeHeaderProps = TypeClassNames
+
+type TypeHeaderProps = TypeGeneralClassNames
 
 const Header = ({ classNames }: TypeHeaderProps) => {
- 
- 
+  const [open, setOpen] = useState<boolean>(false) //Test
+
 
   return (
-    <header className={cn(stls.container, getClassNames({ classNames })) || undefined }>
-        <HeaderTop/>
-        <HeaderMenu/>
-        <HeaderModileMenu/>
+    <header className={cn(stls.container, getClassNames({ classNames })) || undefined}>
+      <Wrapper classNames={[stls.wrapper]}>
+        <GeneralLogo classNames={[stls.iconLogo]}/>
+        <div className={stls.logos}>
+          <ImgGeneralRABOLogo classNames={[stls.iconRABO]} />
```

Comment:
classNames should be renamed to logo
```suggestion
          <ImgGeneralRABOLogo classNames={[stls.logoRABO]} />
```